xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamPVRManager.cpp

```diff
@@ -215,6 +215,24 @@ int CDVDInputStreamPVRManager::GetTime()
   return 0;
 }
 
+bool CDVDInputStreamPVRManager::GetTimes(Times &times)
+{
+  if (m_isRecording)
+    return false;
+
+  PVR_STREAM_TIMES streamTimes;
+  bool ret = CServiceBroker::GetPVRManager().Clients()->GetStreamTimes(streamTimes);
+  if (ret)
```

Comment:
nope, the old functions are complete separate and queried via interface IDisplayTime. The new ones via ITimes. ITimes has preference and IDisplayTime is fallback until it gets removed.